helm-chart/renku-notebooks/templates/statefulset.yaml

```diff
@@ -106,6 +106,22 @@ spec:
               value: {{ .Values.sentry.sampleRate | quote }}
             - name: SENTRY_RELEASE
               value: {{ .Chart.Version | quote }}
+            - name: GIT_CLONE_SENTRY_ENABLED
```

Comment:
@Panaetius we do not need double underscores here. This is just injected into the notebook service container.

The notebook service then uses these environment variables and patches them into the session. Once things are patched into the sessions they need to have the double underscores so that they can be parsed by dataconf.

Here I did not add any double underscores because we do not use dataconf at this level yet.

Reply:
The failing integration tests were a fluke. I reran them locally too without a problem.